bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/pwfs/PwfsRule.java

```diff
@@ -45,14 +45,14 @@ public IP2Problems check(ObservationElements elements)  {
                 try {
                     pwfs.checkBoundaries(primary, ctx).foreach(bs -> {
                         switch (bs) {
-                            case inside:
+                            case nearBoundary:
                                 problems.addWarning(PREFIX + "WARN", "The " + pwfs.getKey() + WARN, targetNode);
                                 break;
-                            case innerBoundary:
+                            case inRange:
                                 // OK, this is where it should be: between inner and outer limits
                                 break;
-                            case outerBoundary:
-                            case outside:
+                            case vignetting:
```

Comment:
Vignetting should trigger a warning, not an error.

Reply:
I thought it was the other way around 🤦‍♂️. I'll fix it.